### ISSUE TITLE
unicode symbol?

### ISSUE THREAD

**Issue by girs1982** [ton-blockchain/ton#939]
Good time,
what is unicode symbol for TON?

**Comment by stenolog** [comment]
No, according to https://unicode-explorer.com/articles/cryptocurrency-unicode-symbols.

**Comment by girs1982** [comment]
> No, according to https://unicode-explorer.com/articles/cryptocurrency-unicode-symbols.
not in this list , they not create unicode icon

**Comment by stenolog** [comment]
@girs1982 note that you can close this issue yourself.

**Comment by girs1982** [comment]
> @girs1982 note that you can close this issue yourself.

we all need unicode symbol , for bots and more , now i fave only ⧨ this

**Comment by stenolog** [comment]
> now i fave only ⧨ this

This one looks nice.

Anyways, you could change the title of the issue to "Provide a TON Unicode Symbol", maybe the leads will take this on.

**Comment by C-o-d-e-C-o-w-b-o-y** [comment]
ꘜ
